### PR TITLE
Update to support Scenic 0.11

### PR DESCRIPTION
**Pull request by axelson**
Unfortunately there's no good upgrade guide for the drivers, but many of the changes are similar to the scene changes:
https://github.com/boydm/scenic/blob/9314020b2962e38bea871e8e1f59cd273dfe0af0/guides/upgrading_to_v0.11.md

I changed the driver to allow Scenic to start the local driver because it feels a bit cleaner to me.

I've also updated the elixir requirement from 1.8 to 1.11 (which is what Scenic 0.11 will require)

Unfortunately there's been an update to the format that hex uses to update mix.lock so the diff there is pretty noisy.

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
```
Comment (axelson) [inline config/config.exs:3]: I also fixed this warning while I was in here

**Comment by lawik** [comment]
Thanks!

**Comment by lawik** [comment]
I figure when @jasonmj is happy with Impression I'll look at cutting a new release of this and Inky.

**Comment by jasonmj** [comment]
@lawik I think the only thing missing from the impression branch on inky is support for the buttons. We were so focused on drawing to the screen, we forgot they were there. :) 

Not sure that's reason enough to block merging the rest of the work.

**Comment by lawik** [comment]
It doesn't have to. @axelson have you looked at the inky work?

I'm bottle-necked by kids so both of you would be more plugged in than me. If y'all think things are ready I'll be happy to take a look and likely just OK it :D

**Pull request merged**